### ISSUE TITLE
Disable reclaim and resource sliders if the respective generator is set to random

### ISSUE THREAD

**Issue by BlackYps** [FAForever/downlords-faf-client#3222]
### Is your feature request related to a problem?

People use the reclaim sliders (for example setting it to low) while keeping the prop generator at random and are then frustrated when a high reclaim map comes out because a prop generator got selected that always produces a lot of reclaim.

### Describe the solution you'd like

We should disable the sliders when the respective generator is set to random.

### Describe alternatives you've considered

ALternatively having the slider restrict which generators can be chosen is the behaviour that we just went away from and it is too opaque for the user.

**Comment by sulikdan** [comment]
@BlackYps Would you have any more description about relations between buttons/sliders? I could try to disable some of them and also add hover-hint message ...

Do I understand correctly: 
Resource Generation set Random -> disable Resource Density slider
Prop Generation set Random -> disable Reclaim Density slider

?

What about Symmetry? There is also random option. Are there any relations? E.g. Fixed seed or even map style?

Anything else?

**Comment by BlackYps** [comment]
yeah, these two that you describe are correct. The symmetry has no implications for the other settings. I don't think we need any other. Some other disabling logic is already implemented and these two seem to be the only ones missing.
